src/plugins/intel_cpu/src/nodes/fullyconnected.h

```diff
@@ -73,12 +83,32 @@ class FullyConnected : public Node {
     ExecutorPtr createExecutor();
     void fuseDecompressionConstant(const MemoryCPtr& memory, MemoryCPtr& decompressionValuesPtr);
 
+    void initTensorParallelConfig(const GraphContext::CPtr context);
+    void needUpdateTensorParalelConfig();
+    void needPrepareParamsForTensorParallel();
+    void initTensorParallelSync();
+    void execTensorParallelSync();
+    void needSplitMemoryForTensorParallel();
+    void needSplitScaleForTensorParallel(const MemoryCPtr& memory);
+    void needUpdateScaleForTensorParallel();
+    void needSplitZeroPointForTensorParallel(const MemoryCPtr& memory);
+    void needUpdateZeroPointForTensorParallel();
+    void needUpdateDQScaleForTensorParallel(std::vector<float>& dequantizationScales);
+
     FCAttrs attrs;
     PostOps postOps;
     MemoryArgs memory;
     ExecutorFactoryPtr<FCAttrs, node::FullyConnected> factory;
     ExecutorPtr executor = nullptr;
     std::string errorPrefix;
+
+    FCTensorParallelConfig tp_cfg;
+    MemoryPtr cached_splited_weight = nullptr;
+    MemoryPtr cached_splited_bias = nullptr;
+    MemoryPtr cached_scale = nullptr;
+    MemoryPtr cached_zeropoint = nullptr;
+    MemoryPtr cached_dst = nullptr;
+    MemoryDescPtr memory_desc;
```

Comment:
Why didn't we include this into a FCTensorParallelConfig?
It seems it is only used in the context of tensor parallel extention.

Reply:
> Why didn't we include this into a FCTensorParallelConfig? It seems it is only used in the context of tensor parallel extention.

yes. move these to FCTensorParallelConfig.